Python/tests/hypergraph_constructors.py

```
@@ -195,24 +213,157 @@ def test_constructor_adjacency_tensor_1(self):
+            [1, 0],
+            [0, 1]]
+        )
+        HG = Hypergraph(incidence_matrix=incidence_matrix)
+        HG.nodes['key'] = ['a', 'b', 'c', 'd']
+        HG.add_node(properties={'key':'e'})
+
+        node_df = pd.DataFrame({'Nodes': [0,1,2,3,4],
+                                'key':['a','b','c','d','e']})
+        assert_frame_equal(node_df, HG.nodes, check_dtype=False)
+
+    def test_add_node_2(self):
```

Ruff (0.8.2)

Redefinition of unused `test_add_node_2` from line 244

(F811)